index.html

```diff
@@ -8558,10 +8560,92 @@ <h3>Graph Containers</h3>
 
   <p>When processing a JSON-LD
     <a data-cite="HTML52/semantics-scripting.html#the-script-element">script element</a>,
-    only the resolved document location of the
-    containing HTML document is used to establish the default <a>base IRI</a> of the enclosed
+    the <a data-cite="HTML52/infrastructure.html#document-base-url">Document Base URL</a>
+    of the containing HTML document,
+    as defined in [[HTML52]],
+    is used to establish the default <a>base IRI</a> of the enclosed
```

Comment:
Can we put this `<base>` related content in it's own PR? I'm not confident that we have consensus as a group that this solves for the hopes/intents expressed in the [TAG's review](https://github.com/w3ctag/design-reviews/issues/312#issuecomment-434689623):
> To that end, we would like to define a JavaScript API for those script elements, to reflect that content back out to DOM.

Once there's group-wide consensus on https://github.com/w3c/json-ld-syntax/issues/23 we can revisit this prose.

Reply:
The TAG response regarding base resolution seemed pretty clear to me:

> we would recommend that, where JSON-LD is embedded in HTML, we should treat it as part of the DOM (and it should be! A valuable part!) and resolve relative URIs to absolute URIs using the element.

However, I'm happy to extract this bit and save it for another PR. It may complicate some of the API and test changes, however.

Reply:
I agree with the TAG review that JSON-LD "should be...A valuable part" of the DOM and have it's own related JS APIs to connect it within the DOM, etc. Once that's explored/"done" then I'd fully expect relative URIs to be resolved in that context.

However, as it stands it's just a data block and therefore "raw text"--which might not even be valid JSON-LD when considering it could be wrapped in HTML comments.

Consequently, we have much more to explore here (which I'm personally excited about that!). Flipping the default on how the URIs are resolved *without* doing that exploration and considering the entirety of the TAG's review would be short sighted and ultimately cause more harm than good--but this can continue on #23. 😃 

At any rate, I'd prefer things not go into the spec text until there's consensus...with apologies for it complicating the tests.

Reply:
And...it seems now that this page has refreshed, you've already done that! 😁 Thanks, @gkellogg!